org.musicbrainz.Picard.json

```diff
@@ -1,56 +1,152 @@
 {
-    "id": "org.musicbrainz.Picard",
+    "app-id": "org.musicbrainz.Picard",
     "runtime": "org.freedesktop.Platform",
     "runtime-version": "1.6",
     "sdk": "org.freedesktop.Sdk",
     "command": "picard",
     "rename-desktop-file": "picard.desktop",
     "rename-icon": "picard",
     "finish-args": [
+        /* X11 + XShm access */
         "--share=ipc",
         "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* OpenGL access */
+        "--device=dri",
+        /* Needs to talk to the network: */
         "--share=network",
+        /* GConf */
+        "--talk-name=org.gnome.GConf",
```

Comment:
GConf doesn't seem to be used any more in Picard 2.0, so we can remove this permission.